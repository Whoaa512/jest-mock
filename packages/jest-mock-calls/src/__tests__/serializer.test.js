describe('serializer', () => {
  it('successful snapshot string', async () => {
    const mock = jest.fn();
    mock('AAA', 'BBB');
    mock('AAA', 'CCC');
    expect(mock).toMatchSnapshot();
  });

  it('successful snapshot object', async () => {
    const mock = jest.fn();
    mock(['X', 'Y', 'Z'], { name: 'yutin' });
    mock(['I', 'J'], { name: 'noknok' });
    expect(mock).toMatchSnapshot();
  });
});
