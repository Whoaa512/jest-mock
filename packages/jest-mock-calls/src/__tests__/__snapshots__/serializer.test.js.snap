// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`serializer successful snapshot object 1`] = `
"
[[X,Y,Z],{name:yutin}]
[[I,J],{name:noknok}]
"
`;

exports[`serializer successful snapshot string 1`] = `
"
[AAA,BBB]
[AAA,CCC]
"
`;
